client/components/charts/ScatterChart.tsx

```
@@ -70,6 +71,55 @@ export function ScatterChart({
+    
+    // 文字ごとに処理
+    for (let i = 0; i < text.length; i++) {
+      const char = text[i];
+
+      // 英字と日本語で文字幅を考慮
+      const charWidth = /[\x00-\x7F]/.test(char) ? alphabetWidth : 1;
+      const charLength = charWidth * annotationFontsize;
+      currentLineLength += charLength;
+
+      if (currentLineLength > annotationLabelWidth) {
+        // 現在の行が最大幅を超えた場合、改行
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)